platform/android/java/lib/src/org/godotengine/godot/utils/DeviceUtils.kt

```diff
@@ -44,9 +44,16 @@ fun isHorizonOSDevice(): Boolean {
 	return "Oculus".equals(Build.BRAND, true)
 }
 
+/**
+ * Returns true if running on Meta Horizon OS.
```

Comment:
Small nitpick here, this should have a comment re PICO :)

Reply:
Ah good catch. Let me fix that :)

Reply:
Done :)